server/api.js

```diff
@@ -1,14 +1,66 @@
+import { request } from "express";
 import { Router } from "express";
 import db from "./db";
 
 import logger from "./utils/logger";
 
 const router = Router();
 
+let requests = [
+	{
+		id: 1,
```

Comment:
can you make the IDs like those I mentioned in the trello card? Instead of 1-5, we have 100, 110, 111, 120, 121, 130, 140

Reply:
@gregdyke please review my code. I have not understood why we need the task **GET /api/laptop_donation/<id> (can return full data as is basic query from db)**.
I have used fake data for the laptop donation, please checkout my implementation and give me some more advice.

Reply:
@gregdyke I have added the api endpoint: Get /api/laptop-donations/:id
Please review my code and leave me some feedback.